src/utils/request-queue.js

```diff
@@ -15,21 +15,25 @@ class RequestQueue {
     this.requestPending = false;
     this.pageUnloading = false;
 
-    this.sendTrackingEvents = (options && options.sendTrackingEvents === false)
-      ? false
-      : true; // Defaults to 'true'
+    this.sendTrackingEvents = (
+      options
+      && options.requestQueue
+      && options.requestQueue.sendTrackingEvents === true
+    ) ? true : false; // Defaults to 'false'
 
     // Mark if page environment is unloading
     helpers.addEventListener('beforeunload', () => {
       this.pageUnloading = true;
     });
 
-    this.send();
+    if (this.sendTrackingEvents) {
+      this.send();
+    }
   }
 
   // Add request to queue to be dispatched
   queue(url, method = 'GET', body) {
-    if (this.sendTrackingEvents && !this.humanity.isBot()) {
+    if (!this.humanity.isBot()) {
```

Comment:
There may be legal implications here that are problematic... we should probably revert this change

Reply:
> There may be legal implications here that are problematic... we should probably revert this change

Resolved